backend/core/src/shard/connector.rs

```diff
@@ -77,65 +73,31 @@ impl ShardConnector {
     }
 
     fn handle_message(&mut self, msg: ShardMessage, ctx: &mut <Self as Actor>::Context) {
-        let ShardMessage { conn_id, payload } = msg;
-
-        // TODO: get `NodeId` for `ShardConnId` and proxy payload to `self.chain`.
-    }
-
-    fn start_frame(&mut self, bytes: &[u8]) {
-        if !self.contbuf.is_empty() {
-            log::error!("Unused continuation buffer");
-            self.contbuf.clear();
+        match msg {
+            ShardMessage::New(ip) => (),
+            ShardMessage::Payload { conn_id, payload } => (),
         }
-        self.continue_frame(bytes);
-    }
 
-    fn continue_frame(&mut self, bytes: &[u8]) {
-        if self.contbuf.len() + bytes.len() <= CONT_BUF_LIMIT {
-            self.contbuf.extend_from_slice(&bytes);
-        } else {
-            log::error!("Continuation buffer overflow");
-            self.contbuf = BytesMut::new();
-        }
-    }
-
-    fn finish_frame(&mut self) -> Bytes {
-        mem::replace(&mut self.contbuf, BytesMut::new()).freeze()
+        // TODO: get `NodeId` for `ShardConnId` and proxy payload to `self.chain`.
```

Comment:
dependent on another PR or can it be fixed in this one?

Reply:
This will land in the next PR :)